### ISSUE TITLE
slow loading, stuttering CD audio and 'sticky' menus

### ISSUE THREAD

**Issue by rester159** [Mazamars312/openfpga-pcengine-cd#33]
I’m having issues with the pc engine core where games are very slow to load, the cd audio is defective (sound fx are fine) and the menus are very slow and get stuck. I’m using a 1tb card with all the entire catalogue of pce cd games on it. SD Card was formatted as ExFAT. Anything I should do?

**Comment by Elrinth** [comment]
What is your sdcard brand and name etc so we can see read/write speeds and stuff like that.
Also did you try 0.1.6?

**Comment by cerebroshell** [comment]
> What is your sdcard brand and name etc so we can see read/write speeds and stuff like that. Also did you try 0.1.6?

I can confirm all of this also happens to me and I'm using a Samsung Evo Select 256gb card.

**Comment by DeerSteak** [comment]
I can confirm it with two different brands of cards that are rated for high speeds:

SanDisk 256GB Ultra microSDXC UHS-I 

TEAMGROUP GO Card 256GB Micro SDXC UHS-I U3 

Both are rated for 100 MB/sec. Both formatted ExFAT.

**Comment by rester159** [comment]
when you mean you can confirm it, do you mean that you had the same issue
with both cards?

**Comment by DeerSteak** [comment]
Yes and when I revert to 0.1.6 with my previous ROM files it’s fixed.

**Comment by rester159** [comment]
Hopefully they will fix it with the next update

**Comment by DeerSteak** [comment]
My guess (only a guess) is that the new seek times do not take into consideration the time it takes to access the BIN sector from the SD card so there's a constant underrun.
